src/css/index.css

```diff
@@ -49,10 +49,6 @@
   padding: 0 16px !important;
 }
 
-.sendbird-message-input .sendbird-message-input-text-field {
-  overflow-y: hidden;
```

Comment:
this was not working?

Reply:
Yes, it doesn't work according to the CSS specificity.
Moreover, this option is for hiding overflowed content. If you style it this way, scrolling becomes impossible.